practiseClient/script.js

```diff
@@ -0,0 +1,52 @@
+
+
+let form = document.querySelector("form");
```

Comment:
This is selecting both buttons so both return the same results. The random page path doesn't seem to be in use to allow the random page button to link to the data.